service/builder/exporters_builder.go

```diff
@@ -232,11 +232,7 @@ func (eb *ExportersBuilder) buildExporter(
 
 	inputDataTypes := exportersInputDataTypes[config]
 	if inputDataTypes == nil {
-		// TODO  https://go.opentelemetry.io/collector/issues/294
-		// Move this validation to config/config.go:validateConfig
```

Comment:
The validation is not needed. In Jaeger we rely on the behavior that does not require registering all rec/exp to the pipeline.